docs/vectors.md

```diff
@@ -80,6 +80,26 @@ if err != nil {
 fmt.Println(searchResult.Hits)
 ```
 
+## Querying with Filters
```

Comment:
Appreciate you adding this, would you throw in this at the end ..

```
## Querying with filters (v2.4.3+)
```

Reply:
good point, thanks!